### DISCUSSION TITLE
How do i remove PDS of my server?

### DISCUSSION THREAD

**Discussion by Mailootje** [bluesky-social/pds#192] [General]
Hi,

How can i remove the PDS from my server?

Im getting this if i visit my server IP:3000

```
This is an AT Protocol Personal Data Server (PDS): https://github.com/bluesky-social/atproto

Most API routes are under /xrpc/
```
![image](https://github.com/user-attachments/assets/c372059c-8b64-4cc4-8ee9-c253932289f6)

**Comment by axfelix** [comment]
This is provided by the Caddy reverse proxy configured on the host in `installer.sh`. You can remove Caddy if you no longer want your server to respond on port 3000. I've also added this note to the readme to clarify that this install script is really intended for a dedicated server: https://github.com/bluesky-social/pds/tree/main?tab=readme-ov-file#installing-on-ubuntu-200422042404-and-debian-111213

Sorry for any confusion here!
